infrastructure/identity-aware-proxy/README.md

```diff
@@ -0,0 +1,38 @@
+# Identity-Aware Proxy Samples
+
+## Running iap_validating_server.py
+
+This sample script runs a simple python web server which validates all GET requests to verify if they're being proxied through Google's Identity-Aware Proxy. This sample depends on and uses the [validate_iap_jwt_from_compute_engine](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/3f5de8c8857784e90935379b63c352c0a5f7f8da/iap/validate_jwt.py#L49) function found in [validate_jwt.py](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/iap/validate_jwt.py). Make sure you install the necessary libraries found in [requirements.txt](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/iap/requirements.txt) by running:
```

Comment:
Can we reference this page so that the reader understands more of why this is needed/required.
https://cloud.google.com/iap/docs/signed-headers-howto

Reply:
Done